### ISSUE TITLE
Define librepo's %{gitrev} from command line

### ISSUE THREAD

**Issue by rholy** [rpm-software-management/ci-dnf-stack#67]
Simplify the scripts (seek for the corresponding "FIXME" in the code).

Blocked by: https://github.com/Tojaj/librepo/pull/61

**Comment by rholy** [comment]
No longer needed since the `make_rpm.sh` script is able to build just a SRPM.
